soh/src/overlays/gamestates/ovl_file_choose/z_file_nameset_PAL.c

```diff
@@ -464,6 +465,7 @@ void FileChoose_DrawNameEntry(GameState* thisx) {
                             dayTime = ((void)0, gSaveContext.dayTime);
                             Sram_InitSave(this);
                             gSaveContext.dayTime = dayTime;
+                            this->prevConfigMode = this->configMode;
```

Comment:
I might be missing something, but are these used anywhere in either of the files?

Reply:
I think I added most of these in an attempt to fix something but it was already used in one place previously. I can't remember if I actually ended up needing to use it to fix the thing I was trying to fix or not.

Reply:
Can confirm I never actually ended up using the prevConfigMode value. However, I may actually end up using it, currently if you go back from name entry, it goes all the way back to file select, but it should probably go back to quest select. But then if we skipped quest select then it should go back to file select. Keeping track of the previous config mode seems like the way to go with that.